### PR TITLE
Add CRUD for Teams

### PR DESCRIPTION
**Pull request by filewalkwithme**
## What
This PR introduces a new CRUD for Teams.

## Why
Currently, if an user wants to create/edit/remove a team, the only way is through the Vulcan API. The lack of an UI for teams makes more difficult for a new user to try and experiment with the platform. **The goal of this PR is to reduce the friction when a new user wants to experiment Vulcan.**

## What's included in this PR
- A new screen which allows the creation, edition and removal of teams.
- A feature toggle which controls whether the new CRUD will be available or not, so Vulcan administrators can control if the feature will be available.

## Demo
teams

### CODE REVIEW

**Review by manelmontilla — APPROVED** [review]
Very good contribution. 
I just added 2 commits to improve a little bit a couple of not very important thinks.
Other than that: LGTM

File: src/teams/teams.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  </div>
+</template>
+
+<script lang="ts">
+// Import modules
+import { Component, Prop, Vue, Watch } from "vue-property-decorator";
+import Home from "./home/home.vue";
+import tokenProvider from "../common/token";
+import loadConfig, { Config } from "../common/config";
+import {
```
[the remark this answers is not in the document]
Comment (filewalkwithme) [reply src/teams/teams.vue]: Thanks!

File: src/teams/home/home.vue [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        </div>
+        <div class="tile is-2 is-vertical"></div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue, Watch } from "vue-property-decorator";
+import VueRouter, { RouteConfig } from "vue-router";
```
[the remark this answers is not in the document]
Comment (filewalkwithme) [reply src/teams/home/home.vue]: Agree!

File: src/index.js [hunk 3]
```diff
@@ -26,7 +26,10 @@ $(document).ready(async () => {
     loadData(client);
 })
 
-function init() {
+async function init() {
+    // get configs
+    const cfg = await common.config();
```
[the remark this answers is not in the document]
Comment (filewalkwithme) [reply src/index.js]: Accepted :D
Thanks!

File: src/teams/router/index.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        path: "/",
+        name: "home",
+        component: () => import("../home/home.vue")
+    },
+];
+
+const router = new VueRouter({
+    mode: "hash",
+    // base: process.env.BASE_URL,
```
[the remark this answers is not in the document]
Comment (filewalkwithme) [reply src/teams/router/index.ts]: Agree, I have deleted it

**Comment by filewalkwithme** [comment]
@manelmontilla Thanks for the commits to this PR! I also rebased it with latest changes from master. This is ready for review again.

**Review by julianvilas — APPROVED** [review]
LGTM. Thanks a lot for the contribution @filewalkwithme!! :)

**Pull request merged**